test/common.dart

```diff
@@ -295,6 +297,16 @@ main() {
       _checkQuery(res, [#b, #d, #f, #g, #h, #i]);
     });
 
+    test('overriden symbols', () {
+      var options = new smoke.QueryOptions(
+          excludeOverriden: true, includeInherited: true, includeMethods: true);
+      var res = smoke.query(L2, options);
+      _checkQuery(res, [#m, #incM, #n]);
+      // Check that the concrete #j is there
+      expect(res[0].isFinal, false);
```

Comment:
So this is failing the tests when I run it through the static version.

Both n and m are coming back as methods and not fields. Mirrors version is fine. Is there something else I need to add in end_to_end_test.dart to make them come up as fields?